apps/champions/lib/champions/config.ex

```diff
@@ -133,4 +135,57 @@ defmodule Champions.Config do
       )
     end)
   end
+
+  def import_dungeon_levels_config() do
+    game_id = Utils.get_game_id(:champions_of_mirra)
+
+    {:ok, dungeon_campaign_json} =
+      Application.app_dir(:champions, "priv/dungeon_campaign.json")
+      |> File.read()
+
+    dungeon_super_campaign =
+      Campaigns.get_super_campaign_by_name_and_game("Dungeon", game_id)
+
+    [dungeon_campaign] = dungeon_super_campaign.campaigns
+
+    supplies = Users.Currencies.get_currency_by_name_and_game("Supplies", game_id)
+
+    Jason.decode!(dungeon_campaign_json, [{:keys, :atoms}])
+    |> Enum.map(fn level ->
+      level
+      |> Map.put(
+        :units,
+        level.characters
+        |> Enum.with_index()
+        |> Enum.map(fn {character, index} ->
+          %{
+            level: level.lineup_level + Enum.random(-level.lineup_level_variance..level.lineup_level_variance),
+            tier: 1,
+            rank: 1,
+            selected: true,
+            slot: index,
+            character_id:
+              Characters.get_character_id_by_name_and_game_id(character, Utils.get_game_id(:champions_of_mirra))
+          }
+        end)
+      )
+      |> Map.put(:game_id, game_id)
+      |> Map.put(:campaign_id, dungeon_campaign.id)
+      |> Map.put(:attempt_cost, [%{currency_id: supplies.id, amount: 1}])
```

Comment:
Should this attempt cost be configurable too?

Reply:
Since it was not provided by the game design document, I figured it was not required and unwanted so as to not clutter up the config. I'll check in with Marco